addons/mail/static/src/models/thread_view/thread_view.js

```diff
@@ -67,6 +67,31 @@ function factory(dependencies) {
         // Private
         //----------------------------------------------------------------------
 
+        /**
+         * @private
+         * @returns {mail.messaging}
+         */
+        _computeMessaging() {
+            return [['link', this.env.messaging]];
+        }
+
+        /**
+         * @private
+         * @returns {String[]}
+         */
+        _computeTextInputSendShortcuts() {
+            if (!this.thread) {
+                return;
+            }
+            const isMailingList = this.thread.model === 'mail.channel' && this.thread.mass_mailing;
+            // The goal here is to target small screen sizes as there is no shortcut
+            // to send a message for a mailing channel with small screen size.
```

Comment:
@seb-odoo I had already mentioned a link why enter is not working in the small screen,

because of this condition
(https://github.com/odoo/odoo/blob/14.0/addons/mail/static/src/components/composer_text_input/composer_text_input.js#L282) 'enter' didn't work to send a message for a small screen.

Thanks..

Reply:
Ok thank you, I remember now but I couldn't find the link again!

I think we probably need to fix this condition instead then. So we would need to remove the `!this.env.messaging.device.isMobile` part from there, and ensure that we do not set `enter` when it is not needed. I don't remember why we added that.